tests/playbooks/nginx-template.yml

```diff
@@ -5,6 +5,7 @@
   roles:
     - ansible-role-nginx
   vars:
+    nginx_debug_output: true
```

Comment:
Great idea setting debug output to true on tests!

Reply:
:)